controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -284,11 +289,13 @@ function execute($request) {
 
 			if ($issue) {
 
+				// set publication date
+				$submission->setDatePublished($this->getData('datePublished') ? $this->getData('datePublished') : Core::getCurrentDate());
```

Comment:
Ah, maybe I should test it first... then I will probably understand... :-) 
Earlier, the date_published was in the published_submissions table and thus if a submission was unassigned from an issue, it would be deleted. And, as I said, I am a little bit worried about the case above: if I first assigned the article to a future issue and then to a real issue, which publication date will this article (in the version 1) have...